.github/workflows/crowdin-download.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           pull_request_title: "New Crowdin translations"
           pull_request_body: "New Crowdin pull request with translations"
           pull_request_base_branch_name: "integration"
```

Comment:
Doesn't this have to be `head_ref` too?

Reply:
Oh yeah I guess that would be convenient! Good call!